src/main/resources/reference.conf

```diff
@@ -11,6 +11,8 @@ publication {
   server.truststore.location = ""
   server.truststore.password = ""
 
+  server.repository-write-interval = 10s
```

Comment:
Currently it's 120s in production. 
Generating 130mb file every 10 seconds consumes way too much space and keeps it busy.